elements/pfe-tabs/demo/demo.css

```diff
@@ -41,118 +31,21 @@ html {
     --pfe-icon--Color: #fff;
 }
 
-[class^="example"],
-section.pfe-l-grid {
-    margin: 32px 0;
-    padding: 0 0;
-}
-
-@media screen and (min-width: 900px) {
-    [class^="example"],
-    section.pfe-l-grid {
-        margin-right: 385px !important;
-    }
-}
-
-
-.example>h2 {
-    text-align: right;
-}
-section>h2 {
-    margin-top: 2em;
-}
-
-.subheading {
-  margin: 0;
-  font-size: 18px;
-  font-weight: bold;
-  font-family: "Arial";
-}
-.inner {
-   margin: 0 0 0 10px;
-   max-width:1140px;
-   padding: 50px 20px;
-}
-.toc {
-    position: fixed;
-    top: 0;
-    right: 0;
-    z-index: 5;
-    width: 250px;
-}
-.toc ul {
-    font-size: 11px;
-    list-style: disc;
-}
-@media screen and (max-width: 1240px) {
-  .toc {
-      display: none;
-  }
-}
-.light-background {
```

Comment:
Can we bring back the light background color so we can see that the default tabs variant of wind does not include background colors? @castastrophe

Reply:
For sure. How about a system color though like lightest? It's just easier to apply using the dropdown.

Reply:
This color was chosen because it's not an PFE system color, so its easy to tell that its not part of the components. Lightest is white, which is what it is now. Lighter could work potentially, but I think the earth variant tabs use lightest, so that could blend in too. I don't super care what the hex value is, just as long as it stands out as not part of the tabs themselves.